app/api/views/briefs.py

```diff
@@ -511,7 +525,7 @@ def get_brief(brief_id):
             brief.data['timeframeConstraints'] = ''
             brief.data['attachments'] = []
     else:
-        brief.data['contactEmail'] = [user.email_address for user in brief.users][0]
+        brief.data['contactEmail'] = current_user.email_address
```

Comment:
This should be the contact email for the brief itself - this is for the "Contact buyer" button on the brief info card. So there may need to be logic here to either use the owner of the brief's email, or the email address added in the team if it is a team brief.

Reply:
Nice pick up.  This one is a little tricky.  I've implemented it so that:
* if the opportunity was created by a user in a team, the contact is the team email address if one exists or the user's email address
* if the opportunity was created prior to a user joining a team, the contacts will be all the user's added to that opportunity or if the user's are in a team, all the team email addresses, or a single team email address (eg. if one of two users is in a team)